clients/client-python/setup.py

```diff
@@ -6,16 +6,33 @@
 from setuptools import find_packages, setup
 
 
+try:
+    with open("README.md") as f:
+        long_description = f.read()
+except FileNotFoundError:
+    long_description = "No README.md file found"
+
 setup(
     name="gravitino",
     description="Python lib/client for Gravitino",
-    version="0.5.0",
-    long_description=open("README.md").read(),
+    version="0.5.0.dev12",
```

Comment:
How do we define this version `0.5.0.dev12`?

Reply:
Here covers all versioning Python packages.
https://packaging.python.org/en/latest/discussions/versioning/

The version number need to be changed each commit Pypi.org.